data/mods/Aftershock/mobs/zombies.json

```diff
@@ -267,5 +267,256 @@
       "FILTHY",
       "LOUDMOVES"
     ]
+  },
+  {
+    "id": "mon_moxie_scavenger",
+    "type": "MONSTER",
+    "name": { "str": "MoxPhore Scavenger" },
+    "description": "A human stands here, towering around 2 meters high.  It's flesh is frozen solid, colored black and blue from frostbite.  It constantly scratches at its flesh with its claw like hands, surveying the land with frozen and foggy eyes.",
+    "default_faction": "moxie",
+    "bodytype": "human",
+    "species": [ "MOXIE" ],
+    "diff": 5,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 100,
+    "speed": 115,
+    "material": [ "flesh" ],
+    "symbol": "A",
+    "color": "red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 4,
+    "dodge": 4,
+    "vision_day": 25,
+    "regenerates": 10,
+    "armor_bullet": 5,
+    "armor_bash": 3,
+    "armor_cut": 5,
+    "harvest": "frost_human",
+    "death_drops": "default_moxie_human_death_drops",
+    "special_attacks": [
+      { "type": "bite", "cooldown": 5 },
+      { "id": "scratch", "damage_max_instance": [ { "damage_type": "cut", "amount": 12 } ] }
+    ],
+    "death_function": [ "NORMAL" ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "GOODHEARING",
+      "SWIMS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_moxie_spartan",
+    "type": "MONSTER",
+    "name": { "str": "MoxPhore Spartan" },
+    "description": "A human stands here, its body bulging and ripped as the tissue is torn, showing bluish muscle, and grey bones.  Ice crystals jut out of its visible wounds as it wanders.",
+    "default_faction": "moxie",
+    "bodytype": "human",
+    "species": [ "MOXIE" ],
+    "diff": 10,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 270,
+    "speed": 70,
+    "material": [ "flesh" ],
+    "symbol": "$",
+    "color": "i_red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 4,
+    "dodge": 4,
+    "vision_day": 40,
+    "regenerates": 15,
+    "armor_bullet": 30,
+    "armor_bash": 15,
+    "armor_cut": 20,
```

Comment:
Do we want to give moxies' weak against heat/fire?  Or are we considering them weak against heat and fire because they have no armor against it compared to their other armors?

Reply:
I believe right now just leaving out armor can work, as I don't want these to be _too_ easy to kill, and I worry giving them weakness to fire will do that. 

However, this is still groundwork, and I am 100% fine with if say, for example with further testing and configuring down the road we find out how they currently are is unsatisfactory then we can try to slap fire weakness on them. I mainly didn't include it because most zombies aren't weak to fire, and these aren't like ice demons or something, just very frostbitten.